src/redux/bookSlice.js

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const bookSlice = createSlice({
+  name: 'Book',
+  initialState: [],
+  reducers: {
+    addBook: (state, action) => {
+      const newBook = {
+        id: new Date(),
+        name: action.payload.book,
+      };
+      state.push(newBook);
+    },
+  },
+  deleteBook: (state, action) => state.filter((item) => item.id === action.payload.id),
+});
+export const { addBook, deleteBook } = bookSlice.actions;
```

Comment:
- [ ] in this file you should also Create and Exports Action Creators. (addBook and removeBook functions)

Reply:
the action type is automatically generated by react tool kit